src/components/Addbook.js

```diff
@@ -7,29 +7,42 @@ const Addbook = () => {
   const dispatch = useDispatch();
   const [title, setTitle] = useState('');
   const [author, setAuthor] = useState('');
+  const [category, setCategory] = useState('');
 
   const fetchTitle = (e) => setTitle(e.target.value);
   const fetchAuthor = (e) => setAuthor(e.target.value);
+  const fetchCategory = (e) => setCategory(e.target.value);
 
-  const addNew = (title, author, e) => {
+  const addNew = (title, author, category, e) => {
     e.preventDefault();
     dispatch(addbook({
       id: uuidv4(),
       title,
       author,
+      category,
     }));
+    setTitle('');
+    setAuthor('');
   };
 
   return (
     <div>
+      <h4 className="h4">ADD NEW BOOK</h4>
```

Comment:
- [ ] [OPTONAL] I highly recommend adding some `margin-bottom` to this section because it is stuck on the bottom
 
![image](https://user-images.githubusercontent.com/75126481/217990888-73e0ed3f-6a1d-48f7-a603-61a88e2d9cb3.png)